pkg/solidity-utils/test/foundry/utils/BaseTest.sol

```diff
@@ -141,4 +141,9 @@ abstract contract BaseTest is Test, GasSnapshot {
 
         return scalingFactor;
     }
+
+    /// @dev Decrease `amount` proportional to `base`; e.g., if base = 100, decrease by 1%; if 1000, 0.1%, etc.
```

Comment:
```suggestion
    /// @dev Returns `amount - amount/base`; e.g., if base = 100, decrease `amount` by 1%; if 1000, 0.1%, etc.
```

Just a suggestion. The decrease is inversely proportional to `base`, and not directly proportional, so It required me some interpretation to understand the function description, but I'm not sure if my suggestion is better.